src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -198,8 +203,18 @@ public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l,
                     string catlink = qRow.Find("a:has(img[class='categ_link'])").First().Attr("href");
                     string cat = ParseUtil.GetArgumentFromQueryString(catlink, "tipus");
                     release.Category = MapTrackerCatToNewznab(cat);
+                    if (seasonep == null)
+                        releases.Add(release);
+        
+                    else
+                    {
+                        Match m = Regex.Match(release.Title, @""+ seasonep + @"\s?$", RegexOptions.IgnoreCase);
```

Comment:
A simple release.Title.Contains(seasonep) should have the same effect.

Reply:
No! (Or if it has the same effect than it is a problem)
seasnoep can be s0x in case of season packs. 
But all episode title from the same season will contains s0x (eg, s0xe01, s0ce02 etc).
It will only match of it is the end of the string or i has a space after s0x

Reply:
ok, didn't see the end of string. This match will only match if Title ends with the sesaonep.
I'll make merge it and make some modifications which should improve the code.